modules/modules_test.go

```diff
@@ -68,7 +72,29 @@ func TestMakeMessage(t *testing.T) {
 		t.Fatalf("Invalid module message class `parameters`")
 	}
 
-	raw, err = MakeMessage(MsgClassStop, nil)
+	// Test parameter decompression
```

Comment:
For clarity, I'd recommend making this its own test, maybe under `TestDecompressedAndMakeMessage` ?